pkcs7_test.go

```diff
@@ -283,7 +282,6 @@ func createTestCertificateByIssuer(name string, issuer *certKeyPair, sigAlg x509
 		issuerKey = priv
 	}
 
-	log.Println("creating cert", name, "issued by", issuerCert.Subject.CommonName, "with sigalg", sigAlg)
```

Comment:
I removed these print statements found through out the test files in this repository since they make it difficult to read the results from `make test`. Print statements used to signify whether a test failed are untouched.